Makefile

```diff
@@ -3,13 +3,19 @@
 
 BASE_IMG_URL := http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2016-03-18/2016-03-18-raspbian-jessie-lite.zip
 BASE_IMG_NAME := $(basename $(notdir $(BASE_IMG_URL))).img
+SPRINGBOARD := root/usr/share/tingbot/springboard.tingapp
 
-build/tingbot-os.deb: $(shell find root)
+build/tingbot-os.deb: $(SPRINGBOARD) $(shell find root)
 	# clean up .DS_Store files
 	find root -name ".DS_Store" -delete
 	mkdir -p build
 	dpkg -b root/ build/tingbot-os.deb
 
+$(SPRINGBOARD):
+	# download springboard and install
+	git clone --depth 1 https://github.com/tingbot/springboard.git dl/springboard
+	cp -r dl/springboard/springboard.tingapp $(SPRINGBOARD)
+
```

Comment:
This will bundle the latest version of springboard, which is bad for release management as the same commit in tingbot-os will build differently depending on where master is on the springboard repo. Would be better to lock down to a specific commit or release of springboard. Can be done with git submodules, `git reset --hard` , but probably my preferred way to do this would be to download a tarball from github - e.g. `https://github.com/tingbot/springboard/tarball/9c8d47acd024f0295a13297ea8c5671eb371bebc`